src/Provider/PostNordProvider.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusPickupPointPlugin\Provider;
+
+use Lsv\PdDk\Client;
+use Lsv\PdDk\Exceptions\MalformedAddressException;
+use Lsv\PdDk\Exceptions\ParcelNotFoundException;
+use Setono\SyliusPickupPointPlugin\Model\PickupPoint;
+use Setono\SyliusPickupPointPlugin\Model\PickupPointInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+final class PostNordProvider implements ProviderInterface
+{
+    use ContainerAwareTrait;
```

Comment:
Inject dependencies instead

Reply:
How is that better that this trait?

Reply:
You should inject the things you need from the container in the class instead of just injecting the container.

Reply:
How would I do the parameter check on line 130?